utils/sl/fake_scons/SCons/Warnings/__init__.py

```diff
@@ -19,7 +19,14 @@
 # SOFTWARE.
 """Fake scons environment shutting up pylint on SCons files"""
 
+class Warning:
```

Comment:
(pylint-redefined-builtin) Redefining built-in 'Warning'
(pylint-redefined-builtin) Redefining built-in 'Warning'
(pylint-too-few-public-methods) Too few public methods (0/2)